test/studies/adventOfCode/2022/day11/bradc/day11-par.chpl

```diff
@@ -0,0 +1,181 @@
+use IO, List, Barriers;
+
+config const numRounds = 20;
+
+// things that define a monkey
+class Monkey {
+  var id: int,                 // an integer ID
+      Items: [0..1] list(int, parSafe=true),  // two lists of its items
```

Comment:
I realize for this parallelization you need the parSafe version of list.  I am a little nervous about it being in the blog because for map we are talking about having a separate distributed (and parallel) map and thus taking the parSafe implementation out of the serial map interface and implementation.  If we do this, then to be consistent we would want to do something similar to list and then this blog post would be out of date.  Not a huge deal, just something to consider.  What would it take to make the list field atomic so only one task at a time can access it?

Reply:
Yeah, at the outset of this series I really didn't want to use features that were on the chopping block for 2.0 and agree that this potentially is (as were some other things I wanted to use here but ended up avoiding like `chpl_task_yield()`).  So I reassured myself by believing that this would be an easy case to update if/when we make that change.  I also think `pop()` is potentially on the chopping block for lists, but have been using it heavily in these posts.   In this case, I didn't want the blog post to convey the message "Although we're a parallel language, we don't do anything to help you with that in our standard library, so you have to roll your own locks around things to use them in parallel."